js/circlepack.js

```diff
@@ -81,36 +98,29 @@ Circlepack.prototype.wrangleData = function(){
   // }
   // levels(vis.data, 0);
 
-  // Reorganize data
   vis.root = d3
-    .hierarchy(vis.data)
-    //.sum(function(d) { return d.size; })
-    //
+    .hierarchy(vis.displayData)
     .sum(function(d) {
-      //console.log("d: " + d.name);
       //return d.size;
       return 10;
     })
     .sort(function(a, b) {
-      //console.log("b: " + b.value + " a:" + a.value);
-      // changes the orientation
-      //return b.value - a.value;
+      b_diff = b.data.diff === undefined ? 0 : b.data.diff
+      a_diff = a.data.diff === undefined ? 0 : a.data.diff
+      return a_diff - b_diff;
```

Comment:
To change the ordering of the circles, this sort function will need to be modified. 

The function(a, b) passed into `.sort()` simply sorts an array of circle nodes (ie. dataverses and datasets), comparing the `diff` value of each circle. To position the circles in a different order, say by `date`, you probably want to write a custom compare function that compares dates, instead of just integers.

For more information, see: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
https://stackoverflow.com/questions/6567941/how-does-sort-function-work-in-javascript-along-with-compare-function